src/main.js

```diff
@@ -150,13 +140,10 @@ Apify.main(async () => {
 
     const zpids = new Set(await Apify.getValue('STATE'));
 
-    Apify.events.on('migrating', async () => {
+    Apify.events.on('persistState', async () => {
```

Comment:
zpids set can get really big, and it's only saved to the KV in case of migration. this can overload the run unnecessarily